### DISCUSSION TITLE
Metadata Googlebot-News, app_id, page_id

### DISCUSSION THREAD

**Discussion by Ahmad-Saleem** [vercel/next.js#57763] [App Router]
How i can add the GoogleBot-News to the metadata api, as i can see we only able to use googleBot 
`robots: {
    index: false,
    follow: true,
    nocache: true,
    googleBot: {
      index: true,
      follow: false,
      noimageindex: true,
      'max-video-preview': -1,
      'max-image-preview': 'large',
      'max-snippet': -1,
    },
  },`

Same for fb:app_id and fb:pages, there is no way to add them

**Comment by Phanx091** [comment]
It appears that next13 does not supports googlebot-news, however I found a work around using custom meta tags using [other](https://nextjs.org/docs/app/api-reference/functions/generate-metadata#metadata-fields) 

For example:

```
    robots: {
      googleBot: {
        index: false,
        follow: false
      }
    },
    other: {
      'googlebot-news': 'noindex'
    }
```

outputs:
```
<meta name="googlebot" content="noindex, nofollow">
<meta name="googlebot-news" content="noindex">
```
